### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by iduartgomez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-rust-GSL [![Build Status](https://api.travis-ci.org/GuillaumeGomez/rust-GSL.png?branch=master)](https://travis-ci.org/GuillaumeGomez/rust-GSL) [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/GuillaumeGomez/rust-GSL?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+rust-GSL [![Build Status](https://api.travis-ci.org/GuillaumeGomez/rust-GSL.png?branch=master)](https://travis-ci.org/GuillaumeGomez/rust-GSL) [![Gitter](https://badges.gitter.im/JoinChat.svg)](https://gitter.im/GuillaumeGomez/rust-GSL?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 ========
 
-A __Rust__ binding for the [GSL library][] (the GNU Scientific Library).
+A __Rust__ binding for the [GSL library] (the GNU Scientific Library).
```
Comment (GuillaumeGomez) [inline README.md]: Are you sure this will work? Because I'm not sure if the whitespace will remove the automatic link to the `()` part or not.

**Comment by GuillaumeGomez** [comment]
I'd prefer you to open another PR for the non-README related changes please.

File: README.md (hunk 1 above)
Comment (iduartgomez) [inline README.md]: yes, you can check it out in my fork

**Comment by iduartgomez** [comment]
sure, will split both

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**